wis2box-management/migrations/v1_0b6_to_v1_0b7/update_wcmp2_identifiers.py

```diff
@@ -0,0 +1,114 @@
+###############################################################################
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+###############################################################################
+
+import argparse
+import json
+import os
+from pathlib import Path
+
+from elasticsearch import Elasticsearch
+
+from wis2box.env import (API_BACKEND_URL, DATADIR, STORAGE_PUBLIC,
+                         STORAGE_SOURCE)
+from wis2box.log import LOGGER, setup_logger
+from wis2box.storage import put_data, delete_data
+from wis2box.util import json_serial, yaml_load
+
+LOG_LEVEL = os.getenv('LOG_LEVEL', 'DEBUG').upper()
+setup_logger(loglevel=LOG_LEVEL)
+
+es_index = 'discovery-metadata'
+maxrecords = 1000
+
+data_mappings_file = Path(DATADIR) / 'data-mappings.yml'
+
+try:
+    with data_mappings_file.open() as fh:
+        DATA_MAPPINGS = yaml_load(fh)
+except FileNotFoundError as err:
+    LOGGER.error(f'Data mappings file not found at {data_mappings_file}')
+    raise err
+
+
+def migrate(dryrun):
+    LOGGER.info('Updating station data in Elasticsearch index')
+    LOGGER.info('Connecting to API ...')
+    try:
+        es = Elasticsearch(API_BACKEND_URL)
+    except Exception as err:
+        LOGGER.error(f'Error connecting to {API_BACKEND_URL}')
+        raise err
+
+    LOGGER.info('Updating discovery metadata')
+    try:
+        res = es.search(index=es_index,
+                        query={'match_all': {}},
+                        size=maxrecords)
```

Comment:
whilst `maxrecords` is large for the metadata do we wnat/need to consider paging?

Reply:
Given this a migration, and the the state of existing installations, we are probably safe without paging.